src/foam/lib/json/OutputJSON.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+foam.INTERFACE({
+  package: 'foam.lib.json',
```

Comment:
Why did you need to convert to a FOAM interface?

Reply:
I get compile error if I do not have foam.lib.json.OutputJSON interface model.

Reply:
AssertionError [ERR_ASSERTION]: Could not find any registered class for foam.lib.json.OutputJSON